browser-tests/childrenFeature.ts

```diff
@@ -86,16 +86,10 @@ test('As a guardian I want to edit the details of my child', async (t) => {
 });
 
 test('As a guardian I want to add and delete a child', async (t) => {
-  const {
-    birthDate,
-    city,
-    postalCode,
-    firstName,
-    lastName,
-    relationship,
-  } = t.ctx.addChild;
+  const { birthDate, city, postalCode, firstName, lastName, relationship } =
+    t.ctx.addChild;
   const newChildName = `${firstName} ${lastName}`;
-  const newChildNameRegepx = new RegExp(`${newChildName} .*`);
+  const newChildNameRegepx = new RegExp(newChildName);
```

Comment:
There is a speling error in this const, it should be newChildNameRegexp and not ..epx

Reply:
Long time no see 🙂 Thanks for spotting this one, fixed 👍

Reply:
Long time indeed! Interesting to follow the development of this project from afar!